contribution_guide.md

```diff
@@ -0,0 +1,36 @@
+# Cotributing to uTensor
+
+Welcome to uTensor contribution document, there are two kinds of pull request:
+  1. implement new feature on uTensor:
+  *  create a new issue and discuss the idea with core maintainers. 
+  *  After making consensus on,  just start the implementation of the feture.
+  
+  2. Extend and fix bugs on an feature : 
+  *  Look and make comment on the issue which you are interested in.
+  *  if you need extra help and context, please contact with core developer for help. 
+
+
+Once you finish implementing a feature or bugfix, please send a Pull Request to
+
+# Developing locally with uTensor
```

Comment:
If you have the circle-ci CLI installed you can build and test with just `circleci build`